### PR TITLE
[NT] Convert charge in pence

### PR DESCRIPTION
**Pull request by likeuwill**
## Checklist:

- [x] It contains only changes required by issue (does not contain other PR)
- [ ] Includes the link to an issue (if apply)
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: app/models/payment.rb [hunk 1]
```diff
@@ -69,7 +69,7 @@ def weekly_transactions
       transaction(day, 7.14)
```
Comment (miqs1992) [inline app/models/payment.rb]: probably also here

File: app/models/payment.rb [hunk 2]
```diff
@@ -79,7 +79,12 @@ def transaction(day, charge)
       vrn: charge_details['vrn'],
       travelDate: day,
       tariffCode: charge_details['tariff_code'],
-      charge: charge
+      charge: amount_in_pence(charge)
```
Comment (radekmorytko) [inline app/models/payment.rb]: What do you think about renaming `charge` to `charge_in_pounds` to increase readability?

File: app/models/payment.rb (hunk 1 above)
Comment (likeuwill) [reply app/models/payment.rb]:
```  # Create single transaction object
  def transaction(day, charge)
    {
      vrn: charge_details['vrn'],
      travelDate: day,
      tariffCode: charge_details['tariff_code'],
      charge: amount_in_pence(charge)
    }
  end```

File: app/models/payment.rb (hunk 2 above)
Comment (miqs1992) [reply app/models/payment.rb]: This may be a good idea

Comment (miqs1992) [reply app/models/payment.rb]: but @radekmorytko did you mean to `chargeInPence`?

**Pull request merged**